api/v1alpha1/database_webhook.go

```diff
@@ -42,65 +44,69 @@ func (r *Database) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 var _ webhook.Defaulter = &Database{}
 
-func instanceSpecDefaulterForCreate(r *Database) {
+func instanceSpecDefaulterForCreate(instance *Instance) {
 
-	if len(r.Spec.Instance.DatabaseNames) == 0 {
-		r.Spec.Instance.DatabaseNames = api.DefaultDatabaseNames
+	if len(instance.DatabaseNames) == 0 {
+		instance.DatabaseNames = api.DefaultDatabaseNames
 	}
 
-	if r.Spec.Instance.TimeZone == "" {
-		r.Spec.Instance.TimeZone = "UTC"
```

Comment:
Same here, many of the Webhooks PR change are showing up here. Let's discuss it offline.